java/http/vertx/deployment-tracing.yaml

```diff
@@ -26,21 +26,24 @@ spec:
             value: my-topic
           - name: SEND_INTERVAL
             value: "1000"
-          - name: JAEGER_SERVICE_NAME
+          - name: MESSAGE_COUNT
+            value: "1000000"
+          - name: OTEL_SERVICE_NAME
             value: java-http-vertx-producer
-          - name: JAEGER_AGENT_HOST
-            value: my-jaeger-agent
-          - name: JAEGER_SAMPLER_TYPE
-            value: const
-          - name: JAEGER_SAMPLER_PARAM
-            value: "1"
+          - name: OTEL_EXPORTER_OTLP_ENDPOINT
+            value: http://my-otlp-endpoint:4317
+          - name: COLLECTOR_OLTP_ENABLED
+            value: "true"
```

Comment:
this env var is meant to be used on the Jaeger backend side to expose the OTLP endpoint. It's also not spelled correctly which highlights it's not having any effects on the examples. Just remove it.

Reply:
Refactored